coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelDataTypeToAvroType.java

```diff
@@ -102,6 +102,10 @@ private static Schema basicSqlTypeToAvroType(BasicSqlType relDataType) {
         return Schema.create(Schema.Type.NULL);
       case ANY:
         return Schema.create(Schema.Type.BYTES);
+      case TIMESTAMP:
+        Schema schema = Schema.create(Schema.Type.LONG);
+        schema.addProp("logicalType", "timestamp");
```

Comment:
Is the `milliseconds` information lost during the conversion since base table schema is `"logicalType" : "timestamp-millis"`?

Reply:
Yes, because in our internal Spark 2.X, if `logicalType == "timestamp"`, Spark will convert the avro type to timestamp. So the logicalType should be `"timestamp"` to be recognized by our internal Spark.